src/test/resources/csv/portfolio.json

```diff
@@ -0,0 +1 @@
+[{"id":166781,"portfolio":"Omnicommerce","portfolioId":"91721","it":{"selectedScenario":{"nbJH_DTP-Helpdesk":"12","nbJH_DTP-Cloud&Connectivity":null}}},{"id":145992,"portfolio":"Finance","portfolioId":"91727","it":{"selectedScenario":{"nbJH_DTP-Helpdesk":null,"nbJH_DTP-Cloud&Connectivity":100}}}]
```

Comment:
make one here with alias and one record without alias please :)

Reply:
Ok, no problem ... need to add a new field because [the spec states](http://avro.apache.org/docs/current/spec.html#names) : 

> The name portion of a fullname, record field names, and enum symbols must:
start with [A-Za-z_]
subsequently contain only [A-Za-z0-9_]

Current fields contain hyphen ...